cmd/build/helmify/static/README.md

```diff
@@ -126,13 +126,15 @@ _See [Exempting Namespaces](https://open-policy-agent.github.io/gatekeeper/websi
+| controllerManager.podSecurityContext           | Security context on pod level manager                                                                                                                                                                                                          | []                                                 |
 | controllerManager.exemptNamespaces            | The exact namespaces to exempt by the admission webhook                                                                                                                                                                                             | `[]`                                                                      |
 | controllerManager.exemptNamespacePrefixes     | The namespace prefixes to exempt by the admission webhook                                                                                                                                                                                           | `[]`                                                                      |
 | controllerManager.hostNetwork                 | Enables controllerManager to be deployed on hostNetwork                                                                                                                                                                                             | `false`                                                                   |
 | controllerManager.dnsPolicy                   | Set the dnsPolicy for controllerManager pods                                                                                                                                                                                                        | `ClusterFirst`                                                            |
-| controllerManager.securityContext             | Security context applied on the container                                                                                                                                                                                                           | `{ "allowPrivilegeEscalation": false, "capabilities": "drop": [all], "readOnlyRootFilesystem": true, "runAsGroup": 999, "runAsNonRoot": true, "runAsUser": 1000 }` |
+| controllerManager.securityContext             | Security context for controllerManager applied on the container                                                                                                                                                                                                           | `{ "allowPrivilegeEscalation": false, "capabilities": "drop": [all], "readOnlyRootFilesystem": true, "runAsGroup": 999, "runAsNonRoot": true, "runAsUser": 1000 }` |
```

Comment:
nit: let's undo this change since it's unrelated to the PR.

Reply:
removed